File: src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/CourseConfigListErrorNotification.java
package fi.aalto.cs.apluscourses.intellij.notifications;

import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;

import com.intellij.notification.Notification;
import com.intellij.notification.NotificationType;
import fi.aalto.cs.apluscourses.services.PluginSettings;

public class CourseConfigListErrorNotification extends Notification {

  /**
   * Constructs a notification that notifies the user of an error about course configs.
   */
  public CourseConfigListErrorNotification() {
    super(
        PluginSettings.A_PLUS,
        getText("notification.CourseConfigListErrorNotification.title"),
        getText("notification.CourseConfigListErrorNotification.content"),
        NotificationType.ERROR);
  }

}

File: src/main/kotlin/fi/aalto/cs/apluscourses/notifications/CourseConfigListErrorNotification.kt
package fi.aalto.cs.apluscourses.notifications

import com.intellij.notification.Notification
import com.intellij.notification.NotificationType
import fi.aalto.cs.apluscourses.MyBundle
import fi.aalto.cs.apluscourses.services.PluginSettings

class CourseConfigListErrorNotification
/**
 * Constructs a notification that notifies the user of an error about course configs.
 */
    : Notification(
    PluginSettings.A_PLUS,
    MyBundle.message("notification.CourseConfigListErrorNotification.title"),
    MyBundle.message("notification.CourseConfigListErrorNotification.content"),
    NotificationType.ERROR
)

